components/collection/utils/timeAgo.ts

```diff
@@ -0,0 +1,29 @@
+export const timeAgo = (timestamp: number): string => {
+  const { $i18n } = useNuxtApp()
+  const { ago, second, seconds, minute, minutes, hour, hours, day, days } = {
+    ago: $i18n.t('time.ago'),
+    second: $i18n.t('time.second'),
+    seconds: $i18n.t('time.seconds'),
+    minute: $i18n.t('time.minute'),
+    minutes: $i18n.t('time.minutes'),
+    hour: $i18n.t('time.hour'),
+    hours: $i18n.t('time.hours'),
+    day: $i18n.t('time.day'),
+    days: $i18n.t('time.days'),
+  }
+  const now = new Date().getTime()
+  const secondsSince = Math.floor((now - timestamp) / 1000)
+  if (secondsSince < 60) {
+    return `${secondsSince} ${secondsSince !== 1 ? seconds : second} ${ago}`
+  }
+  const minutesSince = Math.floor(secondsSince / 60)
+  if (minutesSince < 60) {
+    return `${minutesSince} ${minutesSince !== 1 ? minutes : minute} ${ago}`
+  }
+  const hoursSince = Math.floor(minutesSince / 60)
+  if (hoursSince < 24) {
+    return `${hoursSince} ${hoursSince !== 1 ? hours : hour} ${ago}`
+  }
+  const daysSince = Math.floor(hoursSince / 24)
+  return `${daysSince} ${daysSince !== 1 ? days : day} ${ago}`
```

Comment:
What about `months` and `years`? 
please use `formatDistanceToNow` utils function instead

`import { formatDistanceToNow } from 'date-fns'`

Reply:
it doesn't fit the format in the figma file

let's handle it in a follow up issue ok?

Reply:
Oki, let's make a follow up issue. I guess we could define a custom time format with `date-fns`

Reply:
@Jarsen136  may i have your approval on the PR?